VideoApi/VideoApi.Domain/Room.cs

```diff
@@ -7,9 +7,17 @@
 
 namespace VideoApi.Domain
 {
-    public class Room : Entity<long>
-    {
-        public Room(Guid conferenceId, string label, VirtualCourtRoomType type, bool locked)
+    public abstract class Room : Entity<long>
+    { 
+        public Guid ConferenceId { get; private set; }
+        public string Label { get; protected set; }
+        public VirtualCourtRoomType Type { get; private set; }
+        public RoomStatus Status { get; private set; }
+        public virtual List<RoomParticipant> RoomParticipants { get; }
+        public virtual List<RoomEndpoint> RoomEndpoints { get; }
+        public bool Locked { get; private set; }
```

Comment:
could lock be moved to ConferenceRoom?

Reply:
ConsultationRoom **

Reply:
I asked, and we may have other up-coming rooms (JudicialRooms) which can be locked